src/main.rs

```diff
@@ -38,11 +42,13 @@ fn main() {
     else if matches.is_present("FROM_REVISION") && matches.is_present("TO_REVISION") {
         let from_revision = String::from(matches.value_of("FROM_REVISION").unwrap());
         let to_revision = String::from(matches.value_of("TO_REVISION").unwrap());
+        let to_alias = matches.value_of("to-alias").map_or(to_revision.clone(), String::from);
```

Comment:
Nice - I wasn't aware of the `map_or` function myself. I'm a horrible newbie on Rust actually...

Reply:
I'm new too - just take a lot of advantage of Google and the VS Code RLS integration!